packages/chopsticks/src/plugins/fetch-storage/cli.ts

```diff
@@ -20,36 +18,19 @@ export const cli = (y: Argv) => {
     handler: async (argv) => {
       const config = schema.parse(argv)
       if (!config.endpoint) throw new Error('endpoint is required')
-      const fetchStorageConfig = argv.items as any
-      if (!fetchStorageConfig) throw new Error('fetch-storages items are required')
-      const provider = new WsProvider(config.endpoint, 3_000)
-      const apiPromise = new ApiPromise({ provider })
-      await apiPromise.isReady
+      if (!config.block) throw new Error('block is required')
+      if (!argv.items) throw new Error('fetch-storages items are required')
 
       try {
-        let blockHash: string
-        if (config.block == null) {
-          const lastHdr = await apiPromise.rpc.chain.getHeader()
-          blockHash = lastHdr.hash.toString()
-        } else if (typeof config.block === 'string' && config.block.startsWith('0x')) {
-          blockHash = config.block as string
-        } else if (Number.isInteger(+config.block)) {
-          blockHash = await apiPromise.rpc.chain.getBlockHash(Number(config.block)).then((h) => h.toString())
-        } else {
-          throw new Error(`Invalid block number or hash: ${config.block}`)
-        }
-
-        await fetchStorage({
-          blockHash,
+        await fetchStorages({
+          block: config.block,
+          endpoint: config.endpoint,
           dbPath: config.db ?? 'db.sqlite',
```

Comment:
`config.db` needs to be required and alert users to define the one they have on config file. We're defaulting to `db.sqlite` but some configs may not use that. I would suggest we make db name to be block number. i.e: `db-555.sqlite` so you know when pointing to this db contains snapshots of block 555

Reply:
yeah we can have the default name to be `db-${network_name}-${block_number}.sqlite` and have a log message print something like `Storages are saved at db-acala-12345.sqlite, use '--db=db-acala-12345.sqlite --block=12345' to apply it`